.travis.yml

```diff
@@ -23,41 +22,53 @@ env:
     - PIP_DEPENDENCIES="codecov"
     - EVENT_TYPE='push pull_request'
     - DEBUG=True
+    - CONDA_CHANNELS="astropy"
 
 matrix:
   include:
+
     # Do one build with sphinx-astropy as one of the tests bypasses the auto-
     # installation but we want to make sure that test runs for coverage.
     - os: linux
       env: PYTHON_VERSION=3.6 PIP_DEPENDENCIES='codecov'
     - os: linux
-      env: PYTHON_VERSION=3.6 SPHINX_VERSION='1.5' SETUPTOOLS_VERSION=30
-    - os: linux
-      env: PYTHON_VERSION=3.5 SPHINX_VERSION='1.5' SETUPTOOLS_VERSION=30
-           CONDA_DEPENDENCIES=`echo $CONDA_DEPENDENCIES 'numpydoc<0.9'`"
+      env: PYTHON_VERSION=3.6 SPHINX_VERSION='1.7' SETUPTOOLS_VERSION=30
     - os: linux
       env: PYTHON_VERSION=3.6 PIP_DEPENDENCIES='git+https://github.com/sphinx-doc/sphinx.git#egg=sphinx codecov'
            CONDA_DEPENDENCIES="setuptools cython numpy pytest-cov sphinx-astropy"
            EVENT_TYPE='push pull_request cron'
+
     # Test without installing numpy beforehand to make sure everything works
     # without assuming numpy is already installed
     - os: linux
       env: PYTHON_VERSION=3.6 CONDA_DEPENDENCIES='sphinx-astropy cython pytest-cov'
 
+    # Windows builds - for now we need to install sphinx-astropy with pip since
+    # there is not a recent enough version of graphviz according to conda on Windows
+    - os: windows
+      env: PYTHON_VERSION=3.6
+           CONDA_DEPENDENCIES="setuptools cython numpy pytest-cov"
+           PIP_DEPENDENCIES="codecov sphinx-astropy"
+    - os: windows
+      env: PYTHON_VERSION=3.7
+           CONDA_DEPENDENCIES="setuptools cython numpy pytest-cov"
+           PIP_DEPENDENCIES="codecov sphinx-astropy"
+
     # Test conda's clang
     - os: osx
       env:
-        - PYTHON_VERSION=3.5
+        - PYTHON_VERSION=3.7
```

Comment:
we need python 3.8 builds, too

Reply:
I think we can do that in a separate PR since that's well beyond the scope of this one :)